.github/actions/spelling/expect.txt

```diff
@@ -663,3 +663,4 @@ yyyy
 zoneinfo
 zonepivotgroups
 zzz
+powerapps
```

Comment:
Please add the word in a proper location by dictionary order.

Reply:
Sure